WordPress/Classes/Models/Blog.h

```diff
@@ -60,17 +60,24 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
 @property (nonatomic, strong, readwrite) WPAccount      *jetpackAccount;
 @property (nonatomic, assign, readwrite) BOOL           videoPressEnabled;
 @property (nonatomic, assign, readwrite) BOOL           isMultiAuthor;
-@property (nonatomic, assign, readwrite) BOOL           isJetpack;
+@property (nonatomic, assign, readwrite) BOOL           isHostedAtWPcom;
 @property (nonatomic, strong, readwrite) NSString       *icon;
+/**
+ Stores the username for self hosted sites
+ 
+ @warn For WordPress.com or Jetpack Managed sites this will be nil. Use usernameForSite instead
+ */
+@property (nonatomic, strong,  readwrite) NSString       *username;
```

Comment:
We have discussed this before, he promised he won't do it again :smiley: But seriously, since there is a decision on not aligning things, we should probably fix the ones we come across.